### ISSUE TITLE
Unable to find the PoT (translation template) file in the add-on

### ISSUE THREAD

**Issue by Finnboy94** [aaclause/nvda-OpenAI#51]
I used one of the current PO files as the base of the Finnish translation, of course translating the items again. It seems like there's no nvda.pot translation template file in the add-on. Am I missing something?
